src/Maths/Silk.NET.Maths/Silk.NET.Maths.csproj

```diff
@@ -1,10 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.0;net5.0</TargetFrameworks>
-        <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp3.0' Or '$(TargetFramework)' == 'net5.0'">
+        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.0;netcoreapp5.0</TargetFrameworks>
```

Comment:
I'm against merging these TFM-related csproj changes. Ensure you have the latest .NET 5 preview and that you can build using `dotnet build`. I'd imagine IDEs aren't ready for prime time but the TFM should work regardless.